### PR TITLE
✨ add staked slipstream asset module

### PR DESCRIPTION
**Pull request by Thomas-Smets**

### CODE REVIEW

File: src/asset-modules/Slipstream/StakedSlipstreamAM.sol [hunk 1]
```diff
@@ -0,0 +1,508 @@
...
+import { PoolAddress } from "./libraries/PoolAddress.sol";
+import { ReentrancyGuard } from "../../../lib/solmate/src/utils/ReentrancyGuard.sol";
+import { SafeTransferLib } from "../../../lib/solmate/src/utils/SafeTransferLib.sol";
+import { Strings } from "../../libraries/Strings.sol";
+import { TickMath } from "../UniswapV3/libraries/TickMath.sol";
+
+/**
+ * @title Asset Module for Staked Slipstream Liquidity Positions
+ * @author Pragma Labs
+ * @notice The pricing logic and basic information for Slipstream Slipstream Liquidity Positions.
```
Comment (j-vp) [inline src/asset-modules/Slipstream/StakedSlipstreamAM.sol:26]:
```suggestion
 * @notice The pricing logic and basic information for Staked Slipstream Liquidity Positions.
```

File: src/asset-modules/Slipstream/interfaces/ICLGauge.sol [hunk 2]
```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity 0.8.22;
+
+interface ICLGauge {
+    function decreaseStakedLiquidity(
```
Comment (acollette) [inline src/asset-modules/Slipstream/interfaces/ICLGauge.sol:5]: don't think think decreasedStakedLiquidity and increaseStakedLiquidity are ever used?

Comment (Thomas-Smets) [reply src/asset-modules/Slipstream/interfaces/ICLGauge.sol:5]: Ah no, was initially going to add those function as well, but would become very ugly.

**Review by j-vp — DISMISSED** [review]

**Review by j-vp — APPROVED** [review]

**Review by j-vp — DISMISSED** [review]
Gtg

**Review by j-vp — APPROVED** [review]

**Pull request merged**